mautic/mautic
# TPROD-321 - NotificationBundle - Mobile - Convert PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Sane testing steps of #11908 



- `/s/mobile_notifications` List
  - Does page load and render correctly with and without any notifications present
  - Sorting columns work correctly
  - Filtering work correctly
- `/s/mobile_notifications/new` New
  - Can new notifications be created
- `/s/mobile_notifications/view/{id}` Details
  - Page load/render correctly with no errors
- `/s/mobile_notifications/edit/{id}` Details
  - Can you edit the notification
